Week8/geg222/Mission2/src/pages/MainPage/HomePage.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import SortButtons from "../../components/SortButtons";
+import LpCard from "../../components/LpCard/LpCard";
+import useGetInfiniteLpList from "../../hooks/queries/useGetInfiniteLpList";
+import { useInView } from "react-intersection-observer";
+import LpCardSkeletonList from "../../components/LpCard/LpCardSkeletonList";
+import { LoadingSpinner } from "../../components/LoadingSpinner";
+import LpCreateModal from "../../components/LpCreateModal";
+import useDebounce from "../../hooks/useDebounce";
+import useThrottle from "../../hooks/useThrottle";
+import { SEARCH_DEBOUNCE_DELAY } from "../../constants/delay";
+
+const HomePage = () => {
+  const [search, setSearch] = useState("");
+  const debouncedValue = useDebounce(search, SEARCH_DEBOUNCE_DELAY);
+  const [order, setOrder] = useState<"asc" | "desc">("desc");
```

Comment:
`useGetInfiniteLpList`에서는 `PAGINATION_ORDER` enum을 사용하고 있지만, 
현재 `HomePage`에서는 하드코딩된 "asc" | "desc" 타입을 사용하고 있어 타입 정의가 중복되고 일관성이 떨어집니다. 

enum import를 추가하고 `useState<PAGINATION_ORDER>(PAGINATION_ORDER.DESC)` 같은 형태로 동일한 enum을 사용하면, 타입 안정성을 높이고 코드 유지보수성도 개선할 수 있을 것 같습니다!


```suggestion
  const [order, setOrder] = useState<PAGINATION_ORDER>(PAGINATION_ORDER.desc);
```